packages/common/test/util.js

```diff
@@ -1648,6 +1648,32 @@ describe('common/util', function() {
 
       assert.strictEqual(FakeClass.prototype.methodName, method);
     });
+
+    describe('trailing undefined arguments', function() {
```

Comment:
I think these tests need to go down in the `promisfy` block, this is the `promisifyAll` describe block.